src/elm/Transaction/Event.elm

```diff
@@ -563,17 +566,18 @@ updatePayloadDecoder =
         decode updateType =
             D.field "update" <|
                 case updateType of
-                    "mintDistribution" ->
-                        D.oneOf
-                            [ mintDistributionV1Decoder
-                            , mintDistributionV0Decoder
-                            ]
+                    "mintDistributionV0" ->
+                        mintDistributionV0Decoder
+                            |> D.map MintDistributionPayload
+
+                    "mintDistributionV1" ->
+                        mintDistributionV1Decoder
                             |> D.map MintDistributionPayload
 
                     "transactionFeeDistribution" ->
                         transactionFeeDistributionDecoder |> D.map TransactionFeeDistributionPayload
 
-                    "gASRewards" ->
+                    "gasRewards" ->
                         gasRewardsDecoder |> D.map GasRewardsPayload
 
                     "electionDifficulty" ->
```

Comment:
If you rename `"microGTUPerEuro"` in the rust SDK PR related to this, don't forget to rename it here

Reply:
Renamed